lib/geojsonLayerController.js

```diff
@@ -93,6 +93,8 @@ function processConfig(options) {
     else {
       map.hideGeojsonLayer(name)
     }
+
+    map.map.fire('mappy.data.update', dataService.data)
```

Comment:
This could be map.emit('mappy.data.update', dataService.data) now that the map wrapper is exposed.